app/upgrades/v12/upgrades.go

```diff
@@ -58,9 +65,11 @@ func CreateV12UpgradeHandler(
 			AllowedDenoms:   []string{nativeDenom},
 		}
 		keepers.FeeShareKeeper.SetParams(ctx, newFeeShareParams)
+		logger.Info("set feeshare params")
 
 		// ICA - https://github.com/CosmosContracts/juno/blob/integrate_ica_changes/app/app.go#L846-L885
 		vm[icatypes.ModuleName] = mm.Modules[icatypes.ModuleName].ConsensusVersion()
+		logger.Info("upgraded icatypes version")
```

Comment:
do we need all these logs?

Reply:
I'm going to say yes.

It'll only happen once but will be nice to know.

Reply:
ok